spec/index.html

```diff
@@ -2383,12 +2381,127 @@ <h4>YamlLdErrorCode</h4>
           </section>
         </section>
 
+        <section>
+          <h2>Implementations</h2>
```

Comment:
Implementations are typically listed to the Implementation Report, which should be linked to the spec.

Reply:
Yes, but these implementations are specifically for YAML-LD Extended profile and they do not relate to the body of the spec, do they?

We could also have a separate section for YAML-LD Extended → YAML-LD simple conversion implementations.